CONTRIBUTING.md

```diff
@@ -1,143 +1,177 @@
-- [x] Did you write tests for any new functionality or adapt the existing tests for changed functionality?
-- [x] Are all passwords, credentials and local configuration removed from the code changes?
-- [x] Do all changed files contain some important changes (as opposed to e.g. only whitespace changes)?
-- [x] Is the fork up-to-date with the central repository and can your changes be merged automatically?
-- [x] Did the GitHub Actions CI build run through without test failures?
-
-## Deutsch
-
-### [Verhaltenskodex](https://www.ecamp3.ch/de/verhaltenskodex)
-
-### Git einrichten
-
-Wir wenden einen triangulären Git-Workflow an. Das bedeutet, dass alle Code-Änderungen zuerst auf dem Fork des Entwicklers veröffentlicht werden, und erst dann werden die Änderungen via Pull Request in das offizielle eCamp3-Repository eingefügt. Um dies einzurichten, befolgen wir folgende Schritte:
-
-1. Erstelle einen persönlichen Fork des zentralen Repositories auf GitHub. Nehmen wir an dein GitHub-Account heisst `your-username`.
-
-2. Klone das originale Repository auf deinen lokalen Computer:
-
-   ```bash
-   git clone https://github.com/ecamp/ecamp3.git
-   cd ecamp3
-   ```
-
-3. Füge deinen Fork als Remote hinzu:
-
-   ```bash
-   git remote add your-username https://github.com/your-username/ecamp3.git
-   ```
-
-4. Konfiguriere Git, sodass es als aktuellen, offiziellen Code-Stand das zentrale Repository und fürs Veröffentlichen neuer Änderungen den Fork verwendet:
-
-   ```bash
-   git config remote.pushdefault your-username
-   git config push.default current
-   ```
-
-Wenn dies eingerichtet ist kannst du loslegen, und alle `git pull`-Befehle sollten standardmässig den Code vom zentralen Repository holen und `git push`-Befehle sollten auf deinen eigenen Fork des Projekts senden.
-
-### Ein neues Feature beginnen
-
-Bevor du etwas am Code änderst, solltest du jeweils die folgenden Schritte durchführen, um mit einem sauberen Stand zu starten der später einfach gemerged werden kann:
-
-```bash
-git fetch --all
-git checkout origin/devel
-git checkout -b my-new-feature-branch
+- <details>
+    <summary>run php-cs-fixer and ESLint / Prettier manually before each commit: (Click me, I am Expandable) </summary>
+  
+    ```shell
+    # Frontend fixes in running container
+    docker compose exec frontend npm run lint
+    
+    # API/PHP fixes in running container
+    docker compose exec php composer cs-fix
+    
+    # Print fixes in running container
+    docker compose exec print npm run lint
+    
+    # PDF fixes in running container
+    docker compose exec pdf npm run lint
+    
+    # E2E fixes are always run like this
+    docker compose run --rm --entrypoint="npm run lint" e2e
+    ```
+    If you don't have a container of that type running use 'run' instead of 'execute'. Note that this will start a new Docker container (which might not be desired on a device with limited computing resources).
```

Comment:
I recommend to use run with the --rm flag. Else you have many containers dangling around.

Reply:
Honestly, I just copied and pasted that and added the new containers, I don't use those commands myself. What you are saying makes sense and I will add the --rm flag